docs/source/index.rst

```diff
@@ -23,9 +23,9 @@ Welcome to Kedro's award-winning documentation!
     :target: https://opensource.org/license/apache2-0-php/
     :alt: License is Apache 2.0
 
-.. image:: https://img.shields.io/badge/3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue.svg
+.. image:: https://img.shields.io/badge/3.9%20%7C%203.10%20%7C%203.11-blue.svg
     :target: https://pypi.org/project/kedro/
-    :alt: Python version 3.8, 3.9, 3.10, 3.11
+    :alt: Python version 3.9, 3.10, 3.11
```

Comment:
Just realised we forgot to add 3.12 here. It's not technically part of this issue, so also happy to do that separately 🙂

Reply:
No problem, I can add them in this PR!